source/NoteWindow.cpp

```diff
@@ -176,7 +176,7 @@ NoteWindow :: NoteWindow(entry_ref *ref)
 		// Restore all the references for the NoteView's children
 		fNoteView = new NoteView(msg);
 		fScrollView = (BScrollView*) fNoteView->ChildAt(1);
-		fNoteText = (NoteText*) fScrollView->ChildAt(0);
+		fNoteText = (NoteText*) fScrollView->ChildAt(2);
```

Comment:
Yes, this was the culprit! A `BScrollView` always adds the horizontal/vertical scroll bars (if enabled) before the target view.